### DISCUSSION TITLE
Where is data-table?

### DISCUSSION THREAD

**Discussion by code-dep** [hngngn/shadcn-solid#111] [General]
I used to know that data-table was available.

did you drop the support perhaps?

Thank you!

**Comment by hngngn** [comment]
I will try to bring it back after I'm done with the chart.

**Reply by code-dep** [reply]
ok thank you very much!
In the meantime do you know if It will work if I follow the installation with an older version of the documentation?

**Reply by hngngn** [reply]
yes

**Reply by code-dep** [reply]
thank you!
